src/index.js

```diff
@@ -0,0 +1,324 @@
+const path = require('path');
+const inquirer = require('inquirer');
+
+const Manager = require('./lib/Manager');
+const Engineer = require('./lib/Engineer');
+
+const Intern = require('./lib/Intern');
+const fs = require('fs');
+const generateHTML = (manager, engineers, interns) => {
+  const generateEngineerCard = engineers
```

Comment:
This currently does not generate the engineer cards as it is not a function. Change this to a function in order to generate the cards. 
```suggestion
  const generateEngineerCard = (engineers) => {
      return engineers.map((engineer) => {
          return `...`
      }
  }      
          
      
  
```